### PR TITLE
Issue #1147 Move docs modules to accessible directory

### PR DESCRIPTION
**Pull request by robin-owen**
Per modular documentation guidelines, modules should exist in an
accessible location for potential use in multiple guides.

**Fixes:** Issue #1147

### CODE REVIEW

File: docs/source/topics/proc_starting-monitoring-alerting-telemetry.adoc [hunk 1]
```diff
@@ -20,22 +20,22 @@ For more information, see <<configuring-the-virtual-machine_{context}>>.
 
 ** On Linux or {mac}:
 +
-[subs="+quotes,attributes"]
+[subs="+quotes"]
```
Comment (cfergeau) [inline docs/source/topics/proc_starting-monitoring-alerting-telemetry.adoc:23]: These ",attributes" removals seem unrelated, are they intentional?

Comment (robin-owen) [reply docs/source/topics/proc_starting-monitoring-alerting-telemetry.adoc:23]: Yes, they're intentional, but only because I happened to notice them while building the docs to test how they build with the modules moved around.

The inclusion of "attributes" here causes warnings while building the docs about undefined attributes (the `{end}` portions of the source blocks are interpreted as attributes by AsciiDoctor).

I can move these changes into their own dedicated commit if you'd prefer.

Comment (cfergeau) [reply docs/source/topics/proc_starting-monitoring-alerting-telemetry.adoc:23]: I don't feel that strongly about the separate commit (even if that's better ;), I wanted to make sure this was not an oversight.
